pkgs/servers/authelia/default.nix

```diff
@@ -0,0 +1,110 @@
+{ lib, fetchFromGitHub, buildGoModule, installShellFiles, buildNpmPackage }:
+
+buildGoModule rec {
+  pname = "authelia";
+  version = "4.37.5";
+
+  src = fetchFromGitHub {
+    owner = "authelia";
+    repo = "authelia";
+    rev = "v${version}";
+    sha256 = "sha256-xsdBnyPHFIimhp2rcudWqvVR36WN4vBXbxRmvgqMcDw=";
+  };
+  vendorSha256 = "sha256-mzGE/T/2TT4+7uc2axTqG3aeLMnt1r9Ya7Zj2jIkw/w=";
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  web = buildNpmPackage {
+    inherit src version;
+
+    pname = "authelia-web";
+    sourceRoot = "source/web";
```

Comment:
```suggestion
    pname = "authelia-web";
    inherit version src;

    prePatch = ''
      cd source/web
    '';
```

Reply:
what is the benefit here?

Reply:
Setting `sourceRoot` is too low-level, it isn't suitable to be exposed as if it was an imperative variable.

Reply:
What? I think we do this kind of things in a lot of place, do you have an authoritative source (e.g. recent commits / PRs where we cleaned up this pattern) on that?

Reply:
No.